draws/router.py

```
@@ -1,60 +1,335 @@
+from tickets.db import get_tickets_collection as get_ticket_db_collection
 from rng.utils import calculate_winner_index
 from datetime import datetime
 from xrpl.clients import JsonRpcClient
+# TODO: Find correct import for XRPLClientException for xrpl-py==2.4.0 and reinstate
+# from xrpl.clients import XRPLClientException # This was one attempt
+# from xrpl.exceptions import XRPLClientException # This was another
+# from xrpl import XRPLClientException # And another
 from xrpl.models.requests.ledger import Ledger
 import os
 
+from . import db as draws_db
```

Pylint (3.3.7)

[error] No name 'db' in module ''

(E0611)